app/src/org/commcare/dalvik/dialogs/PaneledChoiceDialog.java

```diff
@@ -89,4 +90,30 @@ public void setOnDismissListener(DialogInterface.OnDismissListener listener) {
     public void dismiss() {
         dialog.dismiss();
     }
+
+    public void addCollapsibleInfoPane(String messageContent) {
+        View extraInfoContainer = view.findViewById(R.id.extra_info_container);
+        extraInfoContainer.setVisibility(View.VISIBLE);
+
+        TextView extraInfoContent = (TextView)view.findViewById(R.id.extra_info_content);
+        extraInfoContent.setText(messageContent);
+
+        final ImageButton extraInfoButton = (ImageButton)view.findViewById(R.id.extra_info_button);
+        extraInfoButton.setVisibility(View.VISIBLE);
+        extraInfoButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                toggleExtraInfoVisibility();
+            }
+        });
+    }
+
+    private void toggleExtraInfoVisibility() {
+        TextView extraInfoContent = (TextView)view.findViewById(R.id.extra_info_content);
```

Comment:
Probably referring to something similar to the ability to expand persistent case tiles to show the case detail. Doing so slides the detail down in an animation.
See https://github.com/dimagi/commcare-odk/blob/master/app/src/org/commcare/android/framework/BreadcrumbBarFragment.java#L182-L204